dotCMS/src/main/webapp/WEB-INF/velocity/preview_mode.vtl

```diff
@@ -137,7 +137,7 @@
 					<iframe id="frameMain" name="frameMain" style="border:0;margin:0;" marginheight="0" marginwidth="0" scrolling="auto" frameborder="0" src="${_baseURI}?mainFrame=1&livePage=$livePage&com.dotmarketing.htmlpage.language=$language&language=$language&host_id=${host.identifier}&$!{queryString}"></iframe>
 				</td>
 				<td>
-					<iframe id="frameMenu" name="frameMenu" style="width:240px;border:0;margin:0;padding:0;" width="240" marginheight="0" marginwidth="0" frameborder="0" scrolling="no" src="${_baseURI}?leftMenu=1&livePage=$livePage&com.dotmarketing.htmlpage.language=$language&language=$language&host_id=${host.identifier}&$!{queryString}"></iframe>
+					<iframe id="frameMenu" name="frameMenu" style="width:240px;border:0;margin:0;padding:0;" width="240" marginheight="0" marginwidth="0" frameborder="0" scrolling="yes" src="${_baseURI}?leftMenu=1&livePage=$livePage&com.dotmarketing.htmlpage.language=$language&language=$language&host_id=${host.identifier}&$!{queryString}"></iframe>
```

Comment:
@lucidmachine looks like **auto** would be a better option here.
```
auto | Scrollbars appear if needed (this is default)
yes  | Scrollbars are always shown (even if they are not needed)
```

Reply:
Fixed in a new PR: https://github.com/dotCMS/core/pull/13217. Closing this one.